.github/README.md

```diff
@@ -42,39 +30,53 @@
 <!-- ABOUT THE PROJECT -->
 ## About The Project
 
-[![Product Name Screen Shot][product-screenshot]](https://example.com)
 
 Breathe AmAIRica is a 3-person group project that required the developers to self-teach a non-explicitly taught technology and build an API-consuming multi-page site in one week.
 We chose Typescript and React Hooks, and to employ these techs by creating and debugging an Air Quality and Weather app for American cities. We were inspired by this recent summer's wildfires, allergies, and the pertinence of air quality and weather data in our climate-changing world. 
 
 Users can view the air quality and weather information for American cities either by the search bar, or by perusing the home page for popular destination cities or cities with positive air quality rankings. Users can view a forecast of the coming week, and click the external links to read in-depth health, safety, and travel articles. 
 
-Time and free API calls were a hinderance, and our app is simply a proof of concept. If we had more time or more access to API data, we would have implemented allergy, pollen, and wildfire API data, and more information about which activities the user can partake in response to the current conditions.
+Time and free API calls were a hinderance, and our app is simply a proof of concept. If we had more time or more access to API data, we would have implemented allergy, pollen, and wildfire API data, and more information about which activities the user can partake in response to the current conditions. We would have also made the page mobile responsive.
 
 ### Built With
-React Hooks
-Typescript
-SCSS
-TDD with Jest
+
+- React Hooks
+- Typescript
+- SCSS
+- TDD with Jest
 
 ## Getting Started
 
 To interact with the app without cloning down the repo, check out our [deployed page]().
 Otherwise, fork and clone down the repo. 
 
-### Prerequisites
-
 Once you have locally cloned down the repo, within that file, run 
 `npm i`
 Once all dependencies are installed, run
 `npm start`
 
-
 ## Usage
 
-Use this space to show useful examples of how a project can be used. Additional screenshots, code examples and demos work well in this space. You may also link to more resources.
+![gif of home](./src/assets/homeload.gif)</br>
+*Home Page*
+
+![gif of location page](./src/assets/locationPage.gif)</br>
+*Location Page*
+
+![gif of about page](./src/assets/about.gif)</br>
+*About Page*
+
+![gif of search functionality](./src/assets/search.gif)</br>
```

Comment:
Cool, I really like this. I will probably be stealing it for future projects.

Reply:
omg I use it all the time